apps/dsp-app/src/app/project/reusable-project-form/reusable-project-form.component.ts

```diff
@@ -65,13 +68,18 @@ export class ReusableProjectFormComponent implements OnInit {
   readonly keywordsValidators = [Validators.minLength(3), Validators.maxLength(64)];
   readonly descriptionValidators = [Validators.minLength(3), Validators.maxLength(40960)];
 
+  get hasChanges(): boolean {
```

Comment:
please don't use this: Angular form knows when the form has changed value.
Something like form.dirty

Reply:
@derschnee68 checking if the form is dirty is not enough, it only check direct fields of the form but not the composite components